maro/simulator/scenarios/finance/account.py

```diff
@@ -0,0 +1,41 @@
+"""Used to maintain stock/futures, one account per episode"""
+from collections import OrderedDict
+
+from maro.backends.frame import NodeAttribute, NodeBase, node
+from maro.simulator.scenarios.finance.common.common import Order, OrderDirection, TradeResult
+
+
+@node("account")
+class Account(NodeBase):
+    remaining_money = NodeAttribute("f")
+    total_money = NodeAttribute("f")
+
+    def __init__(self):
+        self.action_history = OrderedDict()
+
+    def set_init_state(self, init_money: float):
+        self._money = init_money
+        self.remaining_money = self._money
+        self.total_money = self._money
+
+    def take_trade(self, order: Order, trade_result: TradeResult, cur_data: list):
+        if trade_result:
+            if order.direction == OrderDirection.buy:
+                self.remaining_money -= \
```

Comment:
do you need to check the remaining_money is enough or not?

Reply:
checked before operate in account